lib/licensed/dependency.rb

```diff
@@ -68,24 +68,24 @@ def local_files
     # Resets all local project and license information
     def reset_license!
       @project = nil
-      @matched_project_file = nil
+      @project_license = nil
       self.delete("license")
       self.text = nil
     end
 
-    # Returns the Licensee::ProjectFile representing the matched_project_file
-    # or remote_license_file
-    def project_file
-      matched_project_file || remote_license_file
-    end
-
-    # Returns the Licensee::LicenseFile, Licensee::PackageManagerFile, or
-    # Licensee::ReadmeFile with a matched license, in that order or nil
-    # if no license file matched a known license
-    def matched_project_file
-      @matched_project_file ||= project.matched_files
-                                       .select { |f| f.license && !f.license.other? }
-                                       .first
+    # Finds and returns a license from the Licensee::FSProject for this dependency.
+    def project_license
+      @project_license ||= if project.license && !project.license.other?
+        # if Licensee has determined a project to have a specific license, use it!
+        project.license
+      elsif project.license_files.map(&:license).uniq.size > 1
```

Comment:
I _think_ this logic is [already encapsulated in `project.license`](https://github.com/benbalter/licensee/blob/master/lib/licensee/projects/project.rb#L22-L29)?

Reply:
Yea @mlinksva and I were discussing that at https://github.com/github/licensed/pull/64#discussion_r201164703.

TL;DR it's not quite the same due to `project.license` including additional sources over `project.license_files`